tests/integration/data/cassandra/BackendTests.cpp

```diff
@@ -1302,8 +1307,56 @@ TEST_F(BackendCassandraTest, CacheIntegration)
     ASSERT_EQ(done, true);
 }
 
+TEST_F(BackendCassandraTest, CacheFetchLedgerBySeq)
+{
+    runSpawn([&](boost::asio::yield_context yield) {
+        auto rawHeaderBlob = hexStringToBinaryString(kRAWHEADER);
+        ripple::LedgerHeader lgrInfo = util::deserializeHeader(ripple::makeSlice(rawHeaderBlob));
+
+        backend_->writeLedger(lgrInfo, std::move(rawHeaderBlob));
+        auto const testLedgerSeq = lgrInfo.seq;
+        ASSERT_TRUE(backend_->finishWrites(lgrInfo.seq));
```

Comment:
I'm sorry, I don't get it, should this be here?
You overwrite `backend_` few lines below, maybe you should have a separate test class?

Reply:
Made a separate test class